.github/workflows/ms.batch.batchaccounts.yml

```diff
@@ -106,8 +106,7 @@ jobs:
       - name: 'Using test file [${{ matrix.moduleTestFilePaths }}]'
         uses: ./.github/actions/templates/validateModuleDeployment
         with:
-          templateFilePath: '${{ env.modulePath }}/deploy.bicep'
-          parameterFilePath: '${{ env.modulePath }}/${{ matrix.moduleTestFilePaths }}'
+          templateFilePath: '${{ env.modulePath }}/${{ matrix.moduleTestFilePaths }}'
```

Comment:
Just a comment to lock this PR since I guess #1579 should take precedence.

Reply:
Awaiting response from @fblix

Reply:
Response received. This PR should take precedence.